worldeditadditions_commands/commands/selectors/srect.lua

```diff
@@ -0,0 +1,51 @@
+-- ███████ ██████  ███████  ██████ ████████
+-- ██      ██   ██ ██      ██         ██
+-- ███████ ██████  █████   ██         ██
+--      ██ ██   ██ ██      ██         ██
+-- ███████ ██   ██ ███████  ██████    ██
+-- lua parse_params_srect("10")
+-- local
+function parse_params_srect(params_text)
+  local find, _, sn1, ax1, sn2, ax2, len = params_text:find("([+-]?)([xyz]?)%s*([+-]?)([xyz]?)%s*(%d*)")
+
+  -- If ax1 is nil set to player facing dir
+  if ax1 == "" then ax1 = "get"
+  else ax1 = {tonumber(sn1..1),string.lower(ax1)}
+	end
```

Comment:
Some weird formatting / indentation here. Maybe run this file through an automated code formatter? There's a beautifier extension to Atom that might come in handy. Personally my coding style is tabs for indentation.

Reply:
I always use tabs... atom sometimes manages to screw that up though (This wouldn't be the first time that's happened to me)

Reply:
Strange! I sometimes have that problem too. I end up having to run `Whitespace: Convert tabs to spaces` at odd intervals.

Reply:
same here. I wish it would just use tabs exclusively.